code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -356,7 +356,7 @@
 
 /obj/item/weapon/gun/ballistic/automatic/sniper_rifle
 	name = "anti materiel rifle"
```

Comment:
Just a note for the future - need to fix this typo, unless you wanted to quickly fix it up for this pr